src/index.js

```diff
@@ -53,6 +62,7 @@ var RENAMED_COMMANDS = {
 };
 
 program.action(function(cmd, cmd2) {
+  suggestCommands(cmd, commandNames);
```

Comment:
I personally would put this _after_ the `Error... is not a Firebase command`. Then it reads a bit better:

```
» firebase halp
Error: halp is not a Firebase command

Did you mean help?
```